app/src/main/java/com/android/streetworkapp/ui/park/AddEvent.kt

```diff
@@ -107,7 +131,8 @@ fun AddEventScreen(
               }
           FloatingActionButton(
               onClick = {
-                // TODO: use one of the future viewModel to add the event to firebase
+                // TODO: check that it works on the database
+                eventViewModel.addEvent(event)
```

Comment:
Suggestion: to make the data easier to fetch back later and for coherence, i would recommend making having a title for an event mandatory. This can be achieved by modifying the `onClick` callback with something like that:

```onClick = {
                // TODO: check that it works on the database
                  if (event.title.isEmpty()) {
                      Toast.makeText(
                          context,
                          "Please fill the title of the event",
                          Toast.LENGTH_SHORT)
                          .show()
                  } else {
                      eventViewModel.addEvent(event)
                      navigationActions.goBack()
                  }
              } 
```

As well as changing the initial value of the `title` field of your event to an empty string.

Note: context can be obtained by initialising `val context = LocalContext.current` at the start of your composable

Reply:
You are right. I will fix it, thank you!